itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcidTables.java

```diff
@@ -662,4 +662,40 @@ public void testMultiDBTxn() throws Throwable {
     replica.run("drop database " + dbName1 + " cascade");
     replica.run("drop database " + dbName2 + " cascade");
   }
+
+  @Test
+  public void testCtasMmTable() throws Throwable {
+    String tableName = testName.getMethodName();
+    String[] resultArray = new String[]{ "1", "3" };
+    String tableProperties = "\"transactional\"=\"true\", \"transactional_properties\"=\"insert_only\"";
+
+    // Create a non-acid table and replicate it.
+    WarehouseInstance.Tuple bootStrapDump = primary.run("use " + primaryDbName)
+            .run("create table t1_nonacid (a int, b int)")
```

Comment:
i think this test case is already there in test replication acid table incremental load ..just need to change the table property

Reply:
I think, it is better to have new test case here instead of editing incremental acid test suit which is already heavy.

Reply:
Ok, updating current test case and removing new one.